### PR TITLE
feat: vendor fxa-common-password-list and incremental-encoder

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* We want to reduce external dependencies on orphaned code and consolidate libraries.

This commit:

* Adds the `common-password-list` and `incremental-encoder` libraries to the monorepo.

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: libs/vendored/common-password-list/src/scripts/generate-encoded-passwords-module.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  numberOfPasswords
+);
+const encodedPasswords: string[] = encode(passwords);
+
+/*eslint-disable no-console*/
+const output = `/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export default \`${encodedPasswords.join('\n')}\`;
```
[the remark this answers is not in the document]
Comment (bbangert) [reply libs/vendored/common-password-list/src/scripts/generate-encoded-passwords-module.ts:38]: Just legacy really. Was doing fairly minimal changes, can avoid the default here as well though.

**Comment by clouserw** [comment]
image
👀

**Comment by clouserw** [comment]
This is great.  Looking forward to getting these libraries pulled in 👍

File: libs/vendored/common-password-list/src/scripts/generate-encoded-passwords-module.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbangert) [reply libs/vendored/common-password-list/src/scripts/generate-encoded-passwords-module.ts:38]: Removed it for a named export.

**Review by clouserw — APPROVED** [review]

**Pull request merged**